gk/main.c

```diff
@@ -850,6 +850,91 @@ add_ggu_policy(struct ggu_policy *policy,
 	}
 }
 
+static void
+flush_flow_table(struct ip_prefix *src,
+	struct ip_prefix *dst, struct gk_instance *instance)
+{
+	uint16_t proto;
+	uint32_t next = 0;
+	int32_t index;
+	uint32_t num_flushed_flows = 0;
+	const struct ip_flow *key;
+	void *data;
+	struct in_addr ip4_src_mask;
+	struct in_addr ip4_dst_mask;
+	struct in6_addr ip6_src_mask;
+	struct in6_addr ip6_dst_mask;
+
+	memset(&ip6_src_mask, 0, sizeof(ip6_src_mask));
```

Comment:
Add `RTE_VERIFY(src->addr.proto == dst->addr.proto);` just before the `memset()`.